lib/Alchemy/Phrasea/Plugin/ActionBarPluginInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/*
+ * This file is part of Phraseanet
+ *
+ * (c) 2005-2015 Alchemy
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Alchemy\Phrasea\Plugin;
+
+
+interface ActionBarPluginInterface
+{
+
```

Comment:
Each `ActionBarPluginInterface` should implement `public function getActionBar()`